### PR TITLE
Fix tests and upgrade release version

### PR DESCRIPTION
**Pull request by percyperez**
@orcasgit/orcas-developers Please review. Small changes: fix tests and upgrade release version

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,7 @@
+0.2.2 (2016-03-30)
+==================
+* Bug fixes
```
Comment (grokcode) [inline CHANGELOG.rst]: @percyperez I'm thinking it would be nice to be a bit more specific here. "Refresh token bugfixes" maybe.

Comment (percyperez) [inline CHANGELOG.rst]: @grokcode I like that

Comment (percyperez) [inline CHANGELOG.rst]: Fixed!

Comment (brad) [inline CHANGELOG.rst]: :+1:

**Comment by brad** [comment]
Looks good :shipit:

**Comment by grokcode** [comment]
Looks good to me too :rocket:

**Pull request merged**